rust/utils/run-locally/src/main.rs

```diff
@@ -273,11 +272,11 @@ fn main() -> ExitCode {
 
     // Follow-up 'yarn hardhat node' invocation with 'yarn prettier' to fixup
     // formatting on any autogenerated json config files to avoid any diff creation.
-    Command::new("yarn")
-        .args(["prettier"])
-        .current_dir("../")
-        .status()
-        .expect("Failed to run prettier from top level dir");
+    // Command::new("yarn")
+    //     .args(["prettier"])
+    //     .current_dir("../")
+    //     .status()
+    //     .expect("Failed to run prettier from top level dir");
```

Comment:
This should be kept, but I have a change in my PR that makes it only run when you are not in the CI